rest/data/project_test.go

```diff
@@ -444,6 +444,7 @@ func (s *ProjectConnectorGetSuite) TestUpdateProjectVars() {
 
 func TestUpdateProjectVarsByValue(t *testing.T) {
 	require.NoError(t, db.ClearCollections(model.ProjectVarsCollection))
+	require.NoError(t, db.ClearCollections(event.AllLogCollection))
```

Comment:
you can actually just add event.AllLogCollection to line 446

Reply:
done